gcpdiag/lint/gke/snapshots/BP_EXT_2022_001.txt

```diff
@@ -0,0 +1,3 @@
+*  gke/BP_EXT/2022_001: Google Groups for RBAC enabled.
+   (Error: 'authenticatorGroupsConfig')                                   [SKIP]
```

Comment:
This looks weird. What does this error mean?

Reply:
Looks like there was an error with the unit test and it wasn't successfully catching a bug in the main check. This issue should be resolved now.